pkgs/top-level/default.nix

```diff
@@ -19,83 +25,25 @@
                && system != "x86_64-solaris"
                && system != "x86_64-kfreebsd-gnu")
 
```

Comment:
``noSysDirs`` should be in stdenv (true in stdenvNative and false elsewhere), rather than being a top-level argument.